anyway/infographics_utils.py

```diff
@@ -1876,13 +1878,7 @@ def create_infographics_items(request_params: RequestParams) -> Dict:
             "location_text": request_params.location_text,
         }
         output["widgets"] = []
-        output["meta"]["dates_comment"] = (
-            str(request_params.start_time.year)
-            + "-"
-            + str(request_params.end_time.year)
-            + ", עדכון אחרון: "
-            + str(request_params.end_time)
-        )
+        output["meta"]["dates_comment"] = get_dates_comment()
```

Comment:
The `output["meta"]` dict is created at lines 1876-1879. Line 1881 adds a key to this dict. I think it makes sense to add this key as part of the dict creation.